README.md

```diff
@@ -0,0 +1,46 @@
+# External Secrets
+
+This operator reads information from a third party service
+like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)  and automatically injects the values as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) (other secret targets are also planned).
+
+This is a new joint effort to consolidate a single central solution that delivers on most of the requirements gathered from multiple other external secrets projects out there. If you are curious about the origins of this project, check out this issue [kubernetes-external-secrets/issues/47](https://github.com/external-secrets/kubernetes-external-secrets/issues/47) and this [PR](https://github.com/external-secrets/kubernetes-external-secrets/pull/477) and the discussion that triggered this.
+
+
+<a name="features"></a>
```

Comment:
What is the reason behind these HTML markups? 🤔 I don't know how they work 😢

Reply:
It is easier to call it on the TOC, like
``` 
[Planned Features](#features)
```
Not sure if there is a better markdown way.

Reply:
In the GH markdown, you can just refer to all the words on the title.

So if you have a section `# Planned Features` in the TOC you can add `[Planned Features](#planned-features)`